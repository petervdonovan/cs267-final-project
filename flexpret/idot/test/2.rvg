[mu []
[def (core) {t0}]
[def (sender) {t1}]
[def (receiver) {a1}]
[def (noc-base) {t2}]
[def (message) {t6}]
[def (sender-int) 0]
[def (receiver-int) 1]
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [get-coreid core]
    li sender sender-int
    li receiver receiver-int
    li noc-base 0x40000100
    li message 42
    [single-hart {a0} {
      [if+2 [== core sender] {
        sw message [* 4 receiver-int] (noc-base)
      }]
      [delay 0]
      [if+2 [== core receiver] {
        lw t3 [* 4 sender-int] (noc-base)
        [sim-print {t3} {t4}]
      }]}]
    [id die]}]]
]
