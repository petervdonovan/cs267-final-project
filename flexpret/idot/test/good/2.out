Core-1:         42
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
    at Top.scala:126 assert(false.B, "Program terminated sucessfully")
[750] %Error: Top.sim.v:298028: Assertion failed in TOP.Top
%Error: Top.sim.v:298028: Verilog $stop
Aborting...
